docs/source/Support/bskReleaseNotes.rst

```diff
@@ -28,6 +28,9 @@ Version |release|
 -----------------
 - Updated Linux and Windows CI builds to use ``swig`` 4.2.1
 - Updated CI scripts to run on latest macOS and no longer use Ubuntu 20.04
+- Enhanced FSW effector interface modules to zero output messages in their reset methods, ensuring safe management
+  of effector states when algorithms are disabled. This prevents potential runaway operations by clearing stale
+  control values.
```

Comment:
Update the RST documentation of the modules where you are changing behavior.  State now that the `Reset()` command will zero the output messages.